### PR TITLE
Added Light-sheet challenge data

### PR DESCRIPTION
**Pull request by ricardo-re-azevedo**
Updated main page to include the light sheet data as a second challenge dataset. I did some restructuring to the challenge section to make the challenge datasets a subsection. May be good for @lydiang to review as well. I restructured the fMOST section but dont want to overstep.

### CODE REVIEW

File: HCK01_2022_Virtual/README.md [hunk 1]
```diff
@@ -121,9 +238,28 @@ In future hackathons, we will focus on:
 
 [How to add a new reproducible registration processing pipeline?](./ReproducibleResource/README.md)
 
+### Challenge Dataset 1: fMOST Mouse Brain Registration to CCFv3
+
+The fMOST brain volumes and CCFv3 atlas for the hackathon are
+available [on the BIL here](https://download.brainlib.org/hackathon/2022_GYBS/data/). The input is a
+single fMOST NIFTI brain volume.
+
+### Challenge Dataset 1: Light-Sheet Imaged Mouse Brain Registration to CCFv3
```
Comment (thewtex) [inline HCK01_2022_Virtual/README.md:247]: 1 -> 2

File: HCK01_2022_Virtual/README.md [hunk 2]
```diff
@@ -121,9 +238,28 @@ In future hackathons, we will focus on:
...
+
+The Light-sheet imaged brain volumes are available
+[on the BIL here](https://download.brainlib.org/hackathon/2022_GYBS/lightsheet/). For this challenge
+set, the data and atlas is provided for a single brain hemisphere (for size considerations and to
+mimic most real world experiments). Multiple data replicates are provided. We provide one
+dataset `subject0.zarr` in full imaging resolution for those seeking a bigger challenge. As a
+convenience we provide the challenge sets at approximately atlas size as
+Nifti(`subjectN_25.nii. gz`)
```
Comment (thewtex) [inline HCK01_2022_Virtual/README.md:255]: `.nii. gz` -> `.nii.gz`

**Review by thewtex — CHANGES_REQUESTED** [review]

File: HCK01_2022_Virtual/README.md (hunk 1 above)
Comment (ricardo-re-azevedo) [reply HCK01_2022_Virtual/README.md]: resolved in https://github.com/InsightSoftwareConsortium/GetYourBrainStraight/pull/22/commits/fc96c8c7852339bc94e75f55912e2af4ef33702b

File: HCK01_2022_Virtual/README.md (hunk 2 above)
Comment (ricardo-re-azevedo) [reply HCK01_2022_Virtual/README.md]: resolved in https://github.com/InsightSoftwareConsortium/GetYourBrainStraight/commit/fc96c8c7852339bc94e75f55912e2af4ef33702b

**Review by thewtex — APPROVED** [review]
LGTM

**Comment by ricardo-re-azevedo** [comment]
> 

Fixed

**Pull request merged**